test/csuite/schema_abort/main.c

```diff
@@ -793,11 +778,13 @@ thread_run(void *arg)
             switch (__wt_random(&td->data_rnd) % 20) {
             case 0:
                 WT_RELEASE_WRITE_WITH_BARRIER(th_ts[td->info].op, BULK);
-                test_bulk(td);
+                if (!use_txn)
+                    test_bulk(td);
```

Comment:
I think I'd prefer to leave these lines unchanged and just have `test_bulk*` return immediately instead.

Reply:
Done in a26eb92fe7e02d9be96f0ae56d14a0e4c550e03c